test/modules/internet.spec.ts

```diff
@@ -396,9 +397,7 @@ describe('internet', () => {
           expect(username).toBeTruthy();
           expect(username).toBeTypeOf('string');
           expect(username).includes('Aiden');
```

Comment:
Please also check for the lastName specifically as it is easier to understand the check than when it is contained in the regex.

```suggestion
          expect(username).includes('Aiden');
          expect(username).includes('Harann');
```

Reply:
added